src/app/components/audio-recordings/pages/analysis-results/analyses-results.component.ts

```diff
@@ -0,0 +1,193 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { AnalysisJobItemResultsService } from "@baw-api/analysis/analysis-job-item-result.service";
+import { audioRecordingResolvers } from "@baw-api/audio-recording/audio-recordings.service";
+import { projectResolvers } from "@baw-api/project/projects.service";
+import { regionResolvers } from "@baw-api/region/regions.service";
+import { siteResolvers } from "@baw-api/site/sites.service";
+import {
+  audioRecordingMenuItems,
+  audioRecordingsCategory,
+} from "@components/audio-recordings/audio-recording.menus";
+import { PageComponent } from "@helpers/page/pageComponent";
+import { IPageInfo } from "@helpers/page/pageInfo";
+import { AnalysisJob } from "@models/AnalysisJob";
+import { AnalysisJobItemResult } from "@models/AnalysisJobItemResult";
+import { AudioRecording } from "@models/AudioRecording";
+import { Observable, map, takeUntil } from "rxjs";
+
+const audioRecordingKey = "audioRecording";
+const projectKey = "project";
+const regionKey = "region";
+const siteKey = "site";
+
+export const rootPath = "/analysis_jobs/system/results/";
+
+@Component({
+  selector: "baw-analyses-results",
+  templateUrl: "analyses-results.component.html",
+  styleUrls: ["analyses-results.component.scss"],
+})
+export class AnalysesResultsComponent extends PageComponent implements OnInit {
+  public constructor(
+    public api: AnalysisJobItemResultsService,
+    public route: ActivatedRoute
+  ) {
+    super();
+  }
+
+  public rows$: Observable<ResultNode[]>;
+  private readonly routeData = this.route.snapshot.data;
+  public audioRecording: AudioRecording = this.routeData[audioRecordingKey]?.model;
+  private rows = Array<ResultNode>();
+  private rootItemPath: string;
+
+  public get analysisJob(): AnalysisJob {
+    // since this component is currently only used for default analysis jobs, we can hard code the analysis job
+    return new AnalysisJob({
+      id: "system",
+      name: "system",
+    });
+  }
```

Comment:
Since analysis job information is not currently in the route data, I am going to be structuring the new route as follows

`/audio_recordings/:audioRecordingId/analysis_jobs/:analysisJobId/results:path`

Reply:
By using a predefined (hard coded) system analysis job defined in the `analysis-jobs.service.ts`, I was able to get this working with both system jobs, and theoretically user defined jobs.

![image](https://user-images.githubusercontent.com/33742269/211455621-8fd142b2-f3ab-40df-a25d-24913bb0379c.png)